src/main/java/org/influxdb/dto/Point.java

```diff
@@ -83,7 +86,9 @@ public static Builder measurement(final String measurement) {
     public Builder tag(final String tagName, final String value) {
       Preconditions.checkArgument(tagName != null);
       Preconditions.checkArgument(value != null);
-      tags.put(tagName, value);
+      if ( !tagName.isEmpty() && !value.isEmpty()) {
```

Comment:
The first space after ( is breaking the build because this is not conform with the coding style.